### PR TITLE
Clarify the fizzbuzz problem and the workshop task

### PR DESCRIPTION
**Pull request by oliverjam**
Relates #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,11 +15,12 @@ One student starts by creating a single failing test and then the other writes c
 
 ## Defining the problem
 
-Before we start, and for the ones who are not familiar with the FizzBuzz problem, the idea is to display numbers from 1 to 100, but
+Before we start, and for the ones who are not familiar with the [FizzBuzz problem](https://en.wikipedia.org/wiki/Fizz_buzz), the idea is to write a function that takes a number and either:
```
Comment (zurda) [inline README.md]: This really clarifies things, great! 
The only thing is I thought `either` should be followed by an `or` at some point. Is that the case, or can you use it this way as well?

File: README.md [hunk 2]
```diff
@@ -15,11 +15,12 @@ One student starts by creating a single failing test and then the other writes c
...
 
-- for multiples of 3 we will display the word **Fizz** instead of the number,
-- for multiples of 5 we will output **Buzz** instead of the number and
-- for multiples of 3 and 5 we will display **FizzBuzz**.
+- returns the number
+- for multiples of 3 returns the word **Fizz** instead of the number
+- for multiples of 5 returns the word **Buzz** instead of the number
+- for multiples of 3 and 5 returns the word **FizzBuzz**.
```
Comment (zurda) [inline README.md]: It might just be me, but is the order here confusing? I think it's less confusing to start with the exceptions (fizz, buzz, fizzbuzz), and put the `returns the number` at the end since it's for all other cases. Unless you think it's giving the solution away :) What do you think?

File: README.md [hunk 3]
```diff
@@ -171,4 +172,4 @@ Once you have your process nailed:
 - Write a test and watch it fail
 - Write just enough code to pass the test (without breaking any other test that was already passing!)
 
-You can continue working on solving this problem!
+Continue writing tests and refactoring until you have a `fizzbuzz` function that works for any number.
```
Comment (zurda) [inline README.md]: Could we also suggest handling edge cases? For example: `If you have time, try to also handle exceptions/ edge cases? (ie negative numbers, input that isn't a number, zero, etc).`

File: README.md (hunk 1 above)
Comment (oliverjam) [reply README.md]: You're right, to be 100% grammatically correct each bullet point should end with "or", but I felt like that would look weird 🙃

File: README.md (hunk 2 above)
Comment (oliverjam) [reply README.md]: Given that they're supposed to be mutually exclusive bullets ("either/or") the order shouldn't really matter.

File: README.md (hunk 1 above)
Comment (oliverjam) [reply README.md]:
```suggestion
Before we start, and for the ones who are not familiar with the [FizzBuzz problem](https://en.wikipedia.org/wiki/Fizz_buzz), the idea is to write a function that takes a number and either returns the number _or_:
```

and then remove the first bullet

File: README.md [hunk 4]
```diff
@@ -15,11 +15,12 @@ One student starts by creating a single failing test and then the other writes c
 
 ## Defining the problem
 
-Before we start, and for the ones who are not familiar with the FizzBuzz problem, the idea is to display numbers from 1 to 100, but
+Before we start, and for the ones who are not familiar with the [FizzBuzz problem](https://en.wikipedia.org/wiki/Fizz_buzz), the idea is to write a function that takes a number and either:
 
-- for multiples of 3 we will display the word **Fizz** instead of the number,
-- for multiples of 5 we will output **Buzz** instead of the number and
-- for multiples of 3 and 5 we will display **FizzBuzz**.
+- returns the number
```
Comment (oliverjam) [inline README.md]:
```suggestion

```

If we remove this line and add my suggestion for the line above it might make more sense

File: README.md (hunk 3 above)
Comment (oliverjam) [reply README.md]:
```suggestion
Continue writing tests and refactoring until you have a `fizzbuzz` function that works for any number.

If you finish early try writing tests for (and handling) edge-cases (what happens when your function is called with a negative number?)
```

**Comment by zurda** [comment]
LGTM 👍
